nucypher/blockchain/eth/agents.py

```diff
@@ -250,7 +254,10 @@ class StakingEscrowAgent(EthereumContractAgent):
         'finishUpgrade'
     )
 
-    DEFAULT_PAGINATION_SIZE: int = 30    # TODO: Use dynamic pagination size (see #1424)
+    DEFAULT_STAKER_PAGINATION_SIZE_LIGHT_NODE: int = int(os.environ.get(
```

Comment:
Why the cast to `int` here?

Reply:
`os.enrion.get(...)` always returns a string.